standup.py

```diff
@@ -98,6 +98,29 @@ def standup_teams_remove(self, msg, name):
             teams.remove(team)
         return 'Team removed.'
 
+    @arg_botcmd('email', type=str)
+    @arg_botcmd('name', type=str, default=None)
+    def standup_team_configure(self, msg, email, name):
+        """Allows reconfiguring team settings"""
+        team = find_team_from_msg_or_name(msg, name)
+        if not team:
+            return f'Cannot find the team {name}'
+        team.email = email
+        with self.mutable(TEAMS) as teams:
+            for team in teams:
+                if name:
+                    if team.name == name:
+                        break
+                    elif msg.room.frm == team.room:
+                        break
+            else:
+                if name:
+                    return f'Cannot find the team {name}'
+                else:
```

Comment:
no need to else :)

Reply:
If we didn't define name, and attempted to deduce the team from the room name, wouldn't this else return the error message regarding the failure being due to the room, and not the team name?